### PR TITLE
[user-managerd] Initialize pending call pointer. Fixes JB#49867

### PR DESCRIPTION
**Pull request by Tomin1**
Also better logging of user switching.

### CODE REVIEW

File: sailfishusermanager.cpp [hunk 1]
```diff
@@ -528,7 +538,11 @@ void SailfishUserManager::onCreatingJobFailed(SystemdManager::JobList &remaining
             // TODO: What to do?
             qCWarning(lcSUM) << "Unhandled error case";
         }
-    } // else nothing was done
+    } else { // nothing was done
+        qCWarning(lcSUM) << "User switching did not begin";
+        // TODO: This should signal that user switch was cancelled
```
Comment (rainemak) [inline sailfishusermanager.cpp]: Could you log a task and add bug id ref here.

**Comment by rainemak** [comment]
LGTM

**Review by rainemak — APPROVED** [review]
LGTM

**Review by mkenttala — APPROVED** [review]

File: sailfishusermanager.cpp (hunk 1 above)
Comment (Tomin1) [reply sailfishusermanager.cpp]: Yes, created and added.

**Pull request merged**